oteapi/datacache/datacache.py

```diff
@@ -71,27 +66,8 @@ def gethash(
     return hash_.hexdigest()
 
 
-def _asyncrun(func, *args) -> "Any":
-    """Runs `func` in a async thread-pool."""
-    # Adds support for asyncio.
-    # See http://www.grantjenks.com/docs/diskcache/tutorial.html#id13
-    async def async_func(args):
-        loop = asyncio.get_running_loop()
-        future = loop.run_in_executor(None, func, *args)
-        result = await future
-        return result
-
-    return asyncio.run(async_func(args))
-
-
 class DataCache:
-    """Initialize a cache instance with the given download configuration.
-
-    This class is also available to import from `oteapi.datacache`, e.g.:
-
-    ```python
-    from oteapi.datacache import DataCache
-    ```
+    """Initialise a cache instance with the given download configuration.
```

Comment:
```suggestion
    """Initialize a cache instance with the given download configuration.

    This class is also available to import from `oteapi.datacache`, e.g.:

    ```python
    from oteapi.datacache import DataCache
    ```
```

Reply:
Since we are a part of Europe and working in EU projects I perfer the english spelling